docs/src/main/sphinx/connector/kinesis.rst

```diff
@@ -41,6 +41,7 @@ Property Name                           Description
 ``kinesis.aws-region``                  AWS region to be used to read kinesis stream from
 ``kinesis.default-schema``              Default schema name for tables
 ``kinesis.table-description-location``  Directory containing table description files
+``kinesis.table-description-refresh-interval``  Time for identifying the refresh interval for the directory containing table description files
```

Comment:
```suggestion
``kinesis.table-description-refresh-interval``  How often to get the table description from S3
```

Reply:
Please make this table `L36-52` wider. This is one of the CI failure cause. You can build docs by executing `docs/build` locally. If the diff is dirty, I would recommend separating a commit.